FFCDemoPaymentService/Scheduling/SchedulerService.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using FFCDemoPaymentService.Models;
+using FFCDemoPaymentService.Data;
+
+namespace FFCDemoPaymentService.Scheduling
+{
+    public class ScheduleService : IScheduleService
+    {
+        private readonly ApplicationDbContext db;
+        public ScheduleService(ApplicationDbContext db)
+        {
+            this.db = db;
+        }
+
+        public void CreateSchedule(string claimId)
+        {
+            DateTime startDate = DateTime.Now.Date;
+
+            try
+            {
+                db.Schedule.AddRange(ScheduleBuilder(claimId, startDate));
+                db.SaveChanges();
+            }
+            catch(Exception ex)
+            {
+                throw new Exception(ex.Message, ex.InnerException);
```

Comment:
If you're not doing anything with the exception there's not value catching it.  All this does is distort the stack trace.